File: app/src/main/java/com/example/msimple/network/response/CategoryResponse.java
package com.example.msimple.network.response;

import com.example.msimple.data.vos.CategoryVO;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class CategoryResponse {

    @SerializedName("categoriesPrograms")
    List<CategoryVO>categoryVOS;

    public List<CategoryVO> getCategoryVOS() {
        return categoryVOS;
    }
}

File: app/src/main/java/com/example/msimple/network/response/CategoryResponse.kt
package com.example.msimple.network.response

import com.example.msimple.data.vos.CategoryVO
import com.google.gson.annotations.SerializedName

class CategoryResponse {

    @SerializedName("categoriesPrograms")
    var categoryVOS: List<CategoryVO>? = null
        internal set
}
